packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -121,23 +124,19 @@ contract SortedOracles is ISortedOracles, Ownable, Initializable {
   /**
    * @notice Updates an oracle value and the median.
    * @param token The address of the token for which the Celo Gold exchange rate is being reported.
-   * @param numerator The amount of tokens equal to `denominator` Celo Gold.
-   * @param denominator The amount of Celo Gold equal to `numerator` tokens.
+   * @param value The amount of tokens equal to Celo Gold represented as a FixidityLib fraction, premultiplied with the denominator
```

Comment:
The idea of #1365 is not to premultiply with denominator, but instead make users report rates as a fractional number (encoded as a fixidity number).

So if rate is `1`, it should be encoded as `1000000000000000000000000`
and if it's `0.5` it should be `500000000000000000000000`